application/consumer-redis-quarkus/src/test/java/webapp/tier/service/RedisSubscribeServiceTest.java

```diff
@@ -0,0 +1,143 @@
+package webapp.tier.service;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.fppt.jedismock.RedisServer;
+
+import io.quarkus.runtime.ShutdownEvent;
+import io.quarkus.test.junit.QuarkusTest;
+import redis.clients.jedis.Jedis;
+
+@QuarkusTest
+class RedisSubscribeServiceTest {
+
+	@Inject
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5937188)